src/Console/Application.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the webmozart/php-scoper package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Webmozart\PhpScoper\Console;
+
+use Silly\Application as SillyApplication;
+
+class Application extends SillyApplication
+{
+    /**
+     * The version of the PHP-Scoper.
+     */
+    const VERSION = '@package_version@';
```

Comment:
Maybe we could play with https://github.com/Ocramius/example-symfony-cli-usage/pull/4 instead which avoids keeping a hard coded constant in sync